Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ruby:2.4.1-alpine
+
+# Install system dependencies
+RUN apk --update add \
+      build-base \
+      mariadb-dev \
+      postgresql-dev \
+      sqlite-dev \
+      su-exec \
+      tini && \
+    gem update --system && \
+    gem update bundler && \
+    rm -rf /var/cache/apk/*
+
+# Create gemstash user
+ENV GEMSTASH_USER="gemstash"
+ENV GEMSTASH_HOME="/home/${GEMSTASH_USER}"
+RUN addgroup -g "9999" "${GEMSTASH_USER}"
+RUN adduser -S -D -u "9999" -G "${GEMSTASH_USER}" "${GEMSTASH_USER}"
+
+# Install Gemstash
+RUN mkdir -p "${GEMSTASH_HOME}/app"
+WORKDIR "${GEMSTASH_HOME}/app"
+COPY "app/" "${GEMSTASH_HOME}/app"
+RUN bundle config disable_checksum_validation true && \
```

Comment:
Because of [this issue](https://github.com/bundler/bundler/issues/5332). The TL;DR is ... bundler v1.15.x added checksums for gems but it's broken for some gems because the checksums don't match.  The gem that was giving me issues was `faraday_middleware` which is mentioned a lot in that issue.